Winleafs-Server/Winleafs.Server.Api/Configuration/ContextFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.EntityFrameworkCore;
```

Comment:
Startup.cs is not in this PR so quickly commenting here.

CORS is NOT set up correct?

Reply:
It is setup, however not correctly, I think:
````
services.AddCors(options =>
            {
                options.AddPolicy(
                    "AllowSpecificOrigin",
                    builder => builder.WithOrigins("http://localhost:4200")
                        .AllowAnyMethod()
                        .AllowAnyHeader().AllowCredentials());
            });
````

Reply:
CORS shouldn't be set up as we shouldn't allow for websites to use our api, only the winleafs application

Reply:
So what should it be then? AllowAny? Since, in theory, every IP Address running the Winleafs application should be able to connect. Or is there some way of detecting that the Winleafs application is making a request? But a browser needs to make requests as well, for Spotify authorization.

Reply:
Fair enough, didn't know the browser needed to make a request.